ui-v2/app/helpers/format-number.js

```diff
@@ -0,0 +1,19 @@
+import { helper } from '@ember/component/helper';
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
+// the actual helper
+const toLocaleString = function(num) {
+  return num.toLocaleString();
+};
+// wrap this up to help with testing the unit
+export function callIfType(type) {
+  return function(cb) {
+    return function(params, hash) {
+      if (typeof params[0] !== type) {
+        return params[0];
+      }
+      return cb(params[0]);
+    };
+  };
+}
```

Comment:
I see why you're having trouble finding a home for this!

Personally I consider the `/helpers` directory to be sacred and only for helpers. Meanwhile, `/utils` welcomes any ol' code.